algorithms/ar-lscsa/cpp/largest_sum_contiguous_subarray.cpp

```diff
@@ -0,0 +1,47 @@
+#include<bits/stdc++.h>
+using namespace std;
+/*
+Input Format:
+The first and the only argument contains an integer array, A.
+
+Output Format:
+Return an integer representing the maximum possible sum of the contiguous subarray.
+*/
+int maxSumSubArray(const vector<int> &A) {
+    int l=A.size();
+    int curmax=A[0];
+    int ans=A[0];
+    int s=0;

```

Comment:
Codacy found an issue: [Variable 's' is assigned a value that is never used.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6392783)